dotnet/sample-apps/aws-sdk/deploy/wrapper/main.tf

```diff
@@ -12,6 +12,7 @@ module "hello-lambda-function" {
 
   memory_size = 384
   timeout     = 20
+  publish     = true
```

Comment:
What does this change will do?

Reply:
Looks like this parameter is necessary, as the function needs to be published based on this error.

```
 Error: error putting Lambda Provisioned Concurrency Config (lambda-python-amd64:provisioned): InvalidParameterValueException: Provisioned Concurrency Configs cannot be applied to unpublished function versions.
```